include/inoreader_api.h

```diff
@@ -0,0 +1,48 @@
+#ifndef NEWSBOAT_INOREADER_API_H
+#define NEWSBOAT_INOREADER_API_H
```

Comment:
Sorry, these two lines still miss the underscore at the end: it should be `NEWSBOAT_INOREADER_API_H_`.